src/tests/system/tests/test_authselect.py

```diff
@@ -0,0 +1,494 @@
+        2. SSSD started successfully
+        3. Authentication is successful for "user-1"
+    :customerscenario: False
+    :requirement:
+    """
+    client.local.user("user-1").add()
+
+    client.tools.dnf(["remove", "sssd-ldap", "sssd-ad", "sssd-kcm", "sssd-ipa"])
+
+    client.authselect.select("minimal")
+    client.sssd.common.local()
+    client.sssd.start()
+
+
+@pytest.mark.skip
+@pytest.mark.rhel
+@pytest.mark.ticket(bz=1892761)
+@pytest.mark.topology(KnownTopology.Client)
+def test_authselect__when_removed_the_uninstall_is_clean(client: Client):
+    """
+    TODO: Need to test against RHEL.
+    Unable to uninstall Authselect in Fedora because sudo depends on authselect
+
+    :title: Uninstalling authselect will roll back all configuration changes.
+            Authselect makes several critical changes to the authentication stack, this tests to ensures the system
+            is fully reverted and is not broken.
+    :setup:
+        1. Create local user "user-1"
+    :steps:
+        1. Uninstall authselect
+        2. Check authselect configuration files
+        3. Check PAM and NSS configuration files
+        4. Authenticate as "user-1"
+    :expectedresults:
+        1. Authselect is uninstalled
+        2. Authselect configuration directory has been deleted
+        3. PAM and NSS configurations are no longer symbolically linked to authselect files
+        4. Authentication is successful for "user-1"
+    :customerscenario: False
+    :requirement: IDM-SSSD-TC: authselect: Authselect SSSD profile enable and disable mkhomedir
+    """
+    client.local.user("user-1").add()
+
+    client.authselect.select("sssd")
+    client.sssd.common.local()
+    client.sssd.start()
+
+    assert client.auth.ssh.password("user-1", "Secret123")
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_authselect__select_sssd_profile_with_user_auth(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD profile is selected and functionally tested
+    :setup:
+        1. Create POSIX user "user-1"
+    :steps:
+        1. Select SSSD profile
+        2. Start SSSD
+        3. Authenticate as "user-1"
+    :expectedresults:
+        1. SSSD profile is selected
+        2. SSSD is started
+        3. Authentication is successful for "user-1"
+    :customerscenario: False
+    :requirement: IDM-SSSD-TC: authselect: Authselect SSSD profile manual Client configuration
+    """
+    provider.user("user-1").add()
+
+    client.authselect.select("sssd")
+    client.sssd.start()
+
+    assert client.auth.ssh.password("user-1", "Secret123")
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_authselect__sssd_profile_enable_and_disable_mkhomedir(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD profile is selected with mkhomedir feature
+    :setup:
+        1. Create POSIX user "user-1"
+    :steps:
+        1. Select SSSD profile with with-mkhomedir
+        2. Start SSSD
```

Comment:
You are starting SSSD before step 1. Either you change the comment, or you change the code.

Reply:
Same mistake.